lib/tasks/hyacinth.rake

```diff
@@ -3,15 +3,20 @@
 namespace :hyacinth do
   task reindex: :environment do
     DigitalObjectRecord.find_in_batches(batch_size: (ENV['BATCH_SIZE'] || 1000).to_i) do |records|
-      records.each { |record| ::DigitalObject::Base.find(record.uid).index(false) }
+      records.each do |record|
+        ::DigitalObject::Base.find(record.uid).index(false)
+      rescue => e
+        puts "Error while reindexing #{record.uid}. See raised error below:"
+        raise e
+      end
       Hyacinth::Config.digital_object_search_adapter.solr.commit
     end
   end
 
   task purge_all_digital_objects: :environment do
     puts Rainbow("This will delete ALL digital objects in Rails.env #{Rails.env} and cannot be undone. Are you sure you want to do this? (yes/no)").red.bright
     print '> '
-    response = STDIN.gets.chomp
+    response = ENV['yes'] || STDIN.gets.chomp
```

Comment:
I would ticket this separately, but I wonder if the gravity of it might even make having the user verify the name of the environment they are deleting here instead of a simple 'yes'

Reply:
Good call.  For the sake of safety, I'll make that change right now.  Should be easy.  Thanks!